modules/codegen/src/test/scala/aaajunk/JunkTest.scala

```diff
@@ -0,0 +1,52 @@
+package aaajunk
+
+import com.github.javaparser.StaticJavaParser
+import com.github.javaparser.ast.NodeList
+import com.github.javaparser.ast.`type`.UnknownType
+import com.github.javaparser.ast.body.Parameter
+import com.github.javaparser.ast.expr._
+import com.twilio.guardrail.languages.JavaLanguage
+import com.twilio.guardrail.terms.collections._
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.must.Matchers
+
+class JunkTest extends AnyFreeSpec with Matchers {
```

Comment:
Rename this into an entry level test for this strategy and this is great, thank you!

Reply:
Made it more than entry level and even found a typo bug :-P